File: ccc12/J2/main.java
import java.io.*;
import java.util.*;

public class main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int a, b, c, d;

        a = input.nextInt();
        input.nextLine();
        b = input.nextInt();
        input.nextLine();
        c = input.nextInt();
        input.nextLine();
        d = input.nextInt();

        if (a == b && b == c && c == d) {
            System.out.print("Fish At Constant Depth");
        } else if (a < b && b < c && c < d) {
            System.out.print("Fish Rising");
        } else if (a > b && b > c && c > d) {
            System.out.print("Fish Diving");
        } else {
            System.out.print("No Fish");
        }
        input.close();
    }
}

File: ccc12/J2/main.py
import sys
input = sys.stdin.readline
height_a = int(input())
height_b = int(input())
height_c = int(input())
height_d = int(input())

if height_a == height_b and height_b == height_c and height_c == height_d:
    print("Fish At Constant Depth")
elif height_a < height_b and height_b < height_c and height_c < height_d:
    print("Fish Rising")
elif height_a > height_b and height_b > height_c and height_c > height_d:
    print("Fish Diving")
else:
    print("No Fish")
